playbook/app/pb_kits/playbook/pb_map/docs/_map_default.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useRef, useEffect } from 'react'
+import { Map } from '../../'
+
+import maplibregl from 'maplibre-gl'
+
+const MapDefault = () => {
+
+  const mapContainerRef = useRef(null)
+
+    useEffect(() => {
+      if (!maplibregl.supported()) {
+        alert('Your browser does not support MapLibre GL');
+        } else {
+         const map = new maplibregl.Map({
+            container: mapContainerRef.current,
+            style: 'https://basemaps.cartocdn.com/gl/positron-gl-style/style.json',
+            center: [-75.379143, 39.831200],
+            zoom: 13,
+        })
+        //set marker/pin
+        /* eslint-disable-next-line */
+        const marker = new maplibregl.Marker({
+          color: "#0056CF",
+        }).setLngLat([-75.379143, 39.831200])
+        .setPopup(new maplibregl.Popup({className: 'map_popup', closeButton: false}).setHTML(`<h4 class="pb_title_kit_size_4">Hello World!</h4>`)) // add popup
```

Comment:
Let's extract this string into it's own component above....and then we can actually use our kit! I can help if you need more direction on this.